src/templates/resources-page.tsx

```diff
@@ -1,66 +1,109 @@
 import * as React from 'react'
 import * as Page from './page'
+import * as _ from 'lodash'
 import * as PageComponent from '../components/PageComponent'
-const releasePlanImage = require('../../content/images/included/release_plan.png')
-const personaCreationImage = require('../../content/images/included/persona_creation.png')
+import Img from 'gatsby-image'
+import { MarkdownRemark, MarkdownRemarkConnection, ImageSharp } from '../graphql-types'
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  CardHeaderTitle,
+  CardImage,
+  Column,
+  Columns,
+  Container,
+  Section,
+  Title
+} from 'bloomer'
 
-const ContactPageLayout: React.StatelessComponent<Page.PageTemplateLayoutProps> = (props) => {
-  return (
-    <div>
-      <PageComponent.default {...props}>
-        <div className="columns" >
-          <div className="column" >
-            <div className="card">
-              <div className="card-image">
-                <figure className="image is-4x3">
-                  <img src={releasePlanImage} alt="Rlease Plan image" />
-                </figure>
-              </div>
-              <div className="card-content">
-                <div className="media">
-                  <div className="media-content">
-                    <p className="title is-4">Release Plan</p>
-                  </div>
-                </div>
+interface ResourcePageTemplateLayoutProps extends Page.PageTemplateLayoutProps {
+  data?: {
+    imageSharp?: ImageSharp
+    markdownRemark?: MarkdownRemark
+    resource?: MarkdownRemarkConnection
+  },
+}
 
-                <div>
-                  Plan how you can get things in your customer's hands quickly.
-                  Releae planning helps the team understand how we get new stories or features to market.
-              </div>
-              </div>
-            </div>
-          </div>
-          <div className="column" >
-            <div className="card">
-              <div className="card-image">
-                <figure className="image is-4x3">
-                  <img src={personaCreationImage} alt="Persona Creation image" />
-                </figure>
-              </div>
-              <div className="card-content">
-                <div className="media">
-                  <div className="media-content">
-                    <p className="title is-4">Persona Creation</p>
-                  </div>
-                </div>
+const ContactPageLayout: React.StatelessComponent<ResourcePageTemplateLayoutProps> = (props) => {
+  return (
+    <PageComponent.default {...props}>
+      <Container>
```

Comment:
Again, since this is so similar to the Benefits cards, would be nice to have a React component instead of in-lined in page (and essentially copy/pasted from Benefits section on home page)